min/lib/Minify.php

```diff
@@ -494,6 +494,13 @@ protected static function _combineMinify()
             $source = null;
             if ($i < $l) {
                 $source = self::$_controller->sources[$i];
+                
+                //Change the minifier to disable the compression
+                if(!empty(self::$_options['concatOnly'])){
+                    $source->minifier = '';
```

Comment:
Around this line would be better: https://github.com/santoja/minify/blob/master/min/lib/Minify.php#L255